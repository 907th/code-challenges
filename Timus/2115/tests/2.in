4
140 160 150 170
