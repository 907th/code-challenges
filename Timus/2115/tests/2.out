Yes
2 3
